tests/FSharp.Compiler.ComponentTests/CompilerOptions/fsc/warn/warn_level0.fs

```diff
@@ -1,2 +1,16 @@
 // #NoMT #CompilerOptions 
+
+let x = 10
+// Normally a The result of this equality expression has type 'bool' and is implicitly discarded.
+x = 20
+
+let mul x y = x * y
```

Comment:
I don't think this really proves much.
The problem I mentioned on [Slack](https://fsharp.slack.com/archives/C05325FDM/p1686119822502919) was the following:
```fsharp
[<NoComparisson>]
/// Meh
type A = int


let a =
```

This still emits the info on:
![image](https://github.com/dotnet/fsharp/assets/2621499/668f4c47-36e4-4274-8e8c-1b6b570fb97e)

Should the `info` be shown on `--warn:0`? That was the question.

Reply:
I knew I must have missed something, I am an idiot. Will resolve soon...

Reply:
No worries, thanks for looking into this in the first place.